.github/workflows/build.yaml

```diff
@@ -34,7 +34,6 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ needs.bump_version.outputs.tag }}
-          persist-credentials: false
```

Comment:
I think we could keep this line. We do not need git credentials in the next step. And, I think this previously failed because the tag didn't exist.